package.json

```diff
@@ -24,7 +24,8 @@
     "react-dom": "18.3.1"
   },
   "engines": {
-    "node": ">=20"
+    "node": ">=20.15.1 <21",
```

Comment:
If `.nvmrc` is pinned to `20.15.1`, I'd be inclined to pin it here for consistency